model/slider.h

```diff
@@ -32,7 +32,7 @@ namespace minsky
                           [sliderMin,sliderMax] */
 
     mutable double sliderMin, sliderMax, sliderStep;
-    Slider(): sliderBoundsSet(false), sliderStepRel(false) {}
+    Slider(): sliderBoundsSet(true), sliderStepRel(false) {}    // ensure sliderBoundsSet is true upon creation of a variable or parameter. for tickets 1258/1263
```

Comment:
Is the sliderBoundsSet flag ever falsified?

Reply:
No, it's not. Maybe all this requires is a tweak in wiring.tcl, because initSliderBounds runs regardless of whether or not the slider bounds are set the first time?

Reply:
I think the original intention of this flag was to track whether the slider bounds were set (originally variable didn't even display the slider). Perhaps now, we insist that they're set at parameter creation to something sensible, then this flag can be removed.

Reply:
I think the original intention of the flag is very useful, which why I've been striving so hard to try to keep it in my code commits. I think I've now come up with something more reasonable than my previous commit. Your comment about epsilon() was the hint I needed. Thanks.